src/helpers/client.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const client = axios.create({
+  baseURL: "http://127.0.0.1:8000/",
```

Comment:
This url should be an environment variable. You can read more about how to configure one [here](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env)
In this case I would name it something like REACT_APP_API_URL